progress-reports/developer-relations/developer-relations-support-progress-report-mar-2023.md

```diff
@@ -0,0 +1,46 @@
+# Developer Relations - Progress report for Mar/Apr 2023.
+
+## Intro
+This progress report is for the contract work of Fidel Sanchez-Bueno in the Development Relations teams. The report is from  14-03-2023 to 14-04-2023
+
+## Summary
+
+Continued work improving documentation page and tutorial section on icon.community.
+
+Made multiple pull request for fixing issues, reporting bugs and/or making improvements on the different repositories of the ICON Community and ICON Foundation github accounts.
+
+Started test work setting up BTP relayer environment on ICON Testnets.
+
+Collaborating with the drogon team to report issues and improve the product.
+
+Reported issues with drogon:
+* https://github.com/icon-community/drogon/issues/26
+* https://github.com/icon-community/drogon/issues/31
+* https://github.com/icon-community/drogon/issues/32
+
+Collaborating  with the Discord admins for technical support on ICON-related development issues from community members and organizations.
+
+## Deliverables
+
+### Accomplished this cycle
+
+__Deliverables ready__
+| Name | Development State | Notes | Source / location |
+| --------- | ----------------- | ----- | ----------------- |
+| Tutorials on icon.community: How to create a wallet account | Approved |  | [PR](https://github.com/icon-community/icon.community/pull/151) |
+| Tutorials on icon.community: How to check if a validator is missing blocks | Approved |  | [PR](https://github.com/icon-community/icon.community/pull/151)
+| Updated instructions on gochain-local | Approved |  | [PR](https://github.com/icon-project/gochain-local/pull/6) |
+| Created `gochain-local-decentralize`. | Approved | A helper script for decentralizing a local network created using the [gochain-local](https://github.com/icon-project/gochain-local) repo. | [Repo](https://github.com/icon-community/gochain-local-decentralize) |
+| Created "How to Create a Custom Testnet" tutorial for icon.community. | In-Progress |  | [PR](https://github.com/icon-community/icon.community/pull/162) |
```

Comment:
This PR is merged but the development state is in-progress instead of completed. What's left to be done for this?

Reply:
You mean the one about "how to create a custom testnet"? That's a typo it should said completed

Reply:
Yeah, in that case please update it